docs/README.md

```diff
@@ -0,0 +1,328 @@
+# Small_EOD
```

Comment:
>> Czy dokumentacja jest czytelna? GitHub pozwala łatwo komentować.

> Moim zdaniem tak (tylko yaml się nie parsuje poprawnie, warto by wrzucać sobie na podgląd np. tu:https://editor.swagger.io/ przed commitowaniem). Trzymanie dokumentacji w gicie IMHO jest dobrym pomysłem (bo wersjonowanie i komentarze), u nas w projekcie się sprawdza. Sama dokumentacja jest dla mnie jasna, gdybym miała to implementować, to generalnie byłabym zadowolona z tak przygotowanego materiału.

Reply:
Dziękuje za pozytywną recenzje dokumentacji. ;-)

Odnośnie błędów składowych to moje przeoczenie i głównie copy&paste. Poprawiłem. Wprowadziłem automatyczne testowanie ( https://github.com/watchdogpolska/small_eod/pull/70/ ), bo jestem za leniwy, aby kopiować swoje poprawki na jakąś stronę, gdy np. wolontariusz coś poprawi.